tests/locatorsList.ts

```diff
@@ -0,0 +1,34 @@
+import { Locator,Page } from "@playwright/test";
+import { LocatorObject } from "./locatorObject";
+
+export class LocatorsList {
+    private pageModel: Page;
+    private locators: { [key: string]: LocatorObject } = {};
+  
+    constructor(page: Page) {
+      this.pageModel = page;
+    }
+    
+  
+    add(name: string, selector: string): void {
```

Comment:
we usually use this way to insert private element  in the class BUT our selectors used everywhere so please use the same way you used in loginPage.ts to implement the selector in dashboardPage.ts and delete this file

Reply:
ok